README.md

```diff
@@ -60,6 +60,17 @@ If your version of NativeScript is incompatible with the extension you will see
     npm run launch-as-server # launches the debug adapter in server mode
     # execute this in a separate terminal
     npm run test-mac # run tests on ios device
-    ```    
+    ```
 
 4. To install the extension drag and drop the `nativescript-*.*.*.vsix` package in the VS Code.
+
+### How to disable the analytics
```

Comment:
I would recommend using a list for the steps:

The usage data collected by Progress from the NativeScript extension for Visual Studio Code is used strictly to improve the product and its services, and enhance the overall user experience. 

If you have previously enabled the analytics option, you can disable it by following the steps outlined below:

1. Open the Visual Studio Code Settings
    - on Windows, select **File > Preferences > Settings**
    - on macOS, **Code > Preferences > Settings**
2. Add the following option (or update the existing one) to disable the analytics:
```
"nativescript.analytics.enabled": false
```